api/events/schema.py

```diff
@@ -96,16 +97,18 @@ def mutate(self, info, **kwargs):
                 auto_cancelled=True)
             event.save()
         calendar_event = get_single_calendar_event(
-                                                    kwargs['calendar_id'],
-                                                    kwargs['event_id']
-                                                )
+            kwargs['calendar_id'],
+            kwargs['event_id']
+        )
         event_reject_reason = 'after 10 minutes'
         if not notification.event_cancellation_notification(
-                                                            calendar_event,
-                                                            room_id,
-                                                            event_reject_reason
-                                                            ):
+            calendar_event,
+            room_id,
+            event_reject_reason
+        ):
             raise GraphQLError("Event cancelled but email not sent")
+        event_Id=kwargs['event_id']
+        notify.delay(event_Id)
```

Comment:
Please refer to variable naming conventions for `event_Id`. (https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)